MEASURE.md

```diff
@@ -0,0 +1,236 @@
+        } ] 
+      }, ...
+  } ],
+  "type": "summary",
+  "measure": "http://fhir.org/guides/who/anc-cds/Measure/HOUSEHOLDIND01",
+  "date": "2022-06-28T12:28:28+05:00",
+  "period": {
+    "start": "2022-01-01T00:00:00+05:00",
+    "end": "2022-06-28T23:59:59+05:00"
+  },
+  "group": [ {
+      "id": "males",
+      "population": [ {
+          "id": "initial-population",
+          "code": { ... },
+          "count": 136
+        }, {
+          "id": "denominator",
+          "code": { ... },
+          "count": 7
+        }, {
+          "id": "numerator",
+          "code": { ... },
+          "count": 3
+      } ],
+      "measureScore": { "value": 0.42857142857142855 },
+      "stratifier": [ {
+          "id": "by-age",
+          "stratum": [ {
+              "value": { "text": "P0Y" },
+              "population": [ {
+                  "id": "initial-population",
+                  "code": { ... },
+                  "count": 31
+                }, {
+                  "id": "denominator",
+                  "code": { ... },
+                  "count": 4
+                }, {
+                  "id": "numerator",
+                  "code": { ... },
+                  "count": 2
+              } ],
+              "measureScore": { "value": 0.5 }
+            }, { ... ... ... ...}
+              ]
+            }
+          ]
+        }
+      ]
+    }
+  ]
+}
+```
+
+Some notable fields coming/cacluated from Measure
+
+**contained**: The Measure [property](http://hl7.org/fhir/R4/measure-definitions.html#Measure.supplementalData) `supplementalData` is calculated for each measure `subject` and output as an [Observation](http://hl7.org/fhir/R4/observation.html) having extension http://hl7.org/fhir/StructureDefinition/cqf-measureInfo with inner extension defining the variable requested i.e. `group` in case above. The code.coding.code has the value of given variable `Group/1818d503-7226-45cb-9ac7-8c8609dd37c0/_history/3` in example above
+
+**measure**: The Measure.url for which this report was generated i.e. http://fhir.org/guides/who/anc-cds/Measure/HOUSEHOLDIND01
+
+**period**: Measure period which was sent for date filter i.e. reporting period start and end. The Measure interval has closed boundaries. Read details [here](https://cql.hl7.org/02-authorsguide.html#interval-values) 
+
+**group**: The calculated value for each Measure indicator with
+ - id: id/name of group/indicator/stratifier as defined in Meaure. 
+ - count: Calculated value from CQL for given variable
+ - measureScore: The percent/ratio of calculated value i.e. numerator/denomintor. Note that for stratifier the score denominator is stratifier denomintor rather than group denominator
+ - 
+**stratifier**: Count for given indicator disaggregated by each type. The stratifier misses the values where counts are zero. Hence if stratifier has predefined criteria, each should be a calculated as separate group. 
```

Comment:
Good start. I think we can follow perhaps the following flow 
1. Measure  - What a measure and what is it used for 
2. CQL - How we author a CQL script, where to test 
3. How to load CQL Library to FHIR Core  - convert to ELM and load on HAPI, update configs 
4. Invoking $evalute_measure  - How do we call this API on FHIR Core 
5. MeasureReport  - What are the components of interest from a generated MeasureReport
6. Reporting interface on FHIR Core  - How does the reporting interface consume MeasureReport